docs/source/configuration.rst

```diff
@@ -99,4 +99,52 @@ RESPONSE_VALIDATION
     that they conform to the protocol. This should only be used for development
     purposes.
 
+OIDC_PROVIDER
```

Comment:
I would default to 'no' @dcolligan --- if someone comes up with a good reason for broadcasting this information we can always add it in, but I'd rather be over-paranoid first.